prac_09/silver_service_taxi.py

```diff
@@ -0,0 +1,23 @@
+"""
+CP1404/CP5632 Practical - Silver service taxi.
+"""
+from prac_09.taxi import Taxi
+
+
+class SilverServiceTaxi(Taxi):
+    """Represent silver service taxi object."""
+    flagfall = 4.5
+
+    def __init__(self, name, fuel, fanciness):
+        """Initialize silver service taxi object."""
+        super().__init__(name, fuel)
+        self.fanciness = fanciness
+        self.price_per_km *= self.fanciness
+
+    def __str__(self):
+        """Represent a string of SilverServiceTaxi object."""
+        return f"{super().__str__()} plus flagfall of ${self.price_per_km:.2f}"
+
+    def get_fare(self):
+        """Return flagfall plus fare rounded to nearest 10 cent."""
+        return round(self.flagfall + super().get_fare(), 1)
```

Comment:
Good job on the rounding, I missed this in my code 👍

Reply:
Ye though I kinda missed this one too, i should have added it to the Taxi class so it applies to all taxi's but thanks.